src/main/java/net/greatstart/dto/DtoUser.java

```diff
@@ -8,6 +8,7 @@
 
 @Data
 public class DtoUser {
+    private Long id;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "id" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)